schema/deploy/shipping/views.sql

```diff
@@ -487,8 +489,8 @@ create or replace view shipping.incidence_model_observation_v3 as
 
            residence_census_tract,
 
-           encounter_responses.flu_shot,
-           encounter_responses.symptoms,
+           coalesce(encounter_responses.flu_shot,fhir.vaccine) as flu_shot,
+           coalesce(encounter_responses.symptoms,fhir.symptoms) as symptoms,
```

Comment:
In the future we may want to flip the coalesce here to prioritize the FHIR responses.